service/scaladsl/kafka/server/src/test/scala/com/lightbend/lagom/scaladsl/kafka/broker/ScaladslKafkaApiSpec.scala

```diff
@@ -43,7 +43,8 @@ class ScaladslKafkaApiSpec extends WordSpecLike with Matchers with BeforeAndAfte
         "akka.remote.netty.tcp.port" -> "0",
         "akka.remote.netty.tcp.hostname" -> "127.0.0.1",
         "akka.persistence.journal.plugin" -> "akka.persistence.journal.inmem",
-        "akka.persistence.snapshot-store.plugin" -> "akka.persistence.snapshot-store.local"
+        "akka.persistence.snapshot-store.plugin" -> "akka.persistence.snapshot-store.local",
+        "lagom.services.kafka_native" -> s"tcp://localhost:${KafkaLocalServer.DefaultPort}"
```

Comment:
We use `"cas_native" -> s"tcp://localhost:${lagomCassandraPort}/cas_native"` as default value for C* in SBT (but the path segment there seems to be unused). Should this be `"lagom.services.kafka_native" -> s"tcp://localhost:${KafkaLocalServer.DefaultPort}/kafka_native` or could be edit the former to remove the unused path segment?